### PR TITLE
edit category-card to be clicked in hall category-card warp

### PR DESCRIPTION
**Pull request by DwinaTech**

### CODE REVIEW

**Comment by DwinaTech** [comment]
[Make All Category Icon linkable (not just the text)](https://trello.com/c/MxTfs2CW/114-make-all-category-icon-linkable-not-just-the-text)

File: client/src/components/Categories/CategoriesOverview/category.css [hunk 1]
```diff
@@ -6,7 +6,10 @@
     justify-content: space-around;
     flex-wrap: wrap;
     padding-bottom: 100px;
-   
+}
```
Comment (kabaros) [inline client/src/components/Categories/CategoriesOverview/category.css]: you should not change links `a` for everything - just change it for category (make it under some class `.cateogry a` or something)

**Review by kabaros — CHANGES_REQUESTED** [review]

File: client/src/components/Categories/CategoriesOverview/category.css (hunk 1 above)
Comment (kabaros) [reply client/src/components/Categories/CategoriesOverview/category.css]: also update for merge conflicts

Comment (DwinaTech) [reply client/src/components/Categories/CategoriesOverview/category.css]: I do not understand what do you mean!!

**Review by kabaros — APPROVED** [review]

**Pull request merged**